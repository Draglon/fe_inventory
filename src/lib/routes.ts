// ---User (SELF)---
export const homeRoute = "/";
export const loginRoute = "/login";
export const registrationRoute = "/registration";
