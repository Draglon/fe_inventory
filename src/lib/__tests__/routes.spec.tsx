import * as routes from "../routes";

it("routes constants match snapshot", () => {
  expect(routes).toMatchSnapshot();
});
