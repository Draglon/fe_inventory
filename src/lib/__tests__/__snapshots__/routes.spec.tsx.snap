// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`routes constants match snapshot 1`] = `
{
  "homeRoute": "/",
  "loginRoute": "/login",
  "registrationRoute": "/registration",
}
`;
